fem/estimators.hpp

```diff
@@ -104,16 +105,19 @@ class ZienkiewiczZhuEstimator : public AnisotropicErrorEstimator
        @param sol      The solution field whose error is to be estimated.
        @param flux_fes The ZienkiewiczZhuEstimator assumes ownership of this
                        FiniteElementSpace and will call its Update() method when
-                       needed. */
+                       needed.
+       @param with_coeff Takes value of 1 if we want to consider the coefficient
+               in the bilinear integrator to weight the fluxed. */
    ZienkiewiczZhuEstimator(BilinearFormIntegrator &integ, GridFunction &sol,
-                           FiniteElementSpace *flux_fes)
+                           FiniteElementSpace *flux_fes, bool with_coeff = false)
```

Comment:
For consistency with the other options (`anisotropic` and `flux_averaging`), it seems more natural for the constructor to NOT handle the option `_with_coeff` either. The user will need to call the `SetWithCoeff` method explicitly, similar to `SetAnisotropic` and `SetFluxAveraging`. I think this approach makes the user code easier to read, even though it is a little more verbose.